doc/source/ray-core/examples/automl_for_time_series.ipynb

```diff
@@ -12,7 +12,15 @@
    "metadata": {},
    "source": [
     "```{tip}\n",
-    "We strongly recommend using [Ray AIR Tuner](air-tuner) for hyperparameter tuning/AutoML, which will enable you to build it faster and more easily, and get the built-in benefits like logging, fault tolerance and many more. If you think your use case cannot be supported by Ray AIR, we'd love to get your feedback e.g. through a [Ray GitHub issue](https://github.com/ray-project/ray/issues).\n",
+    "Ray provides multiple ways to implement this workload. As a general rule, for simple, naively parallel workloads like this one:\n",
+    "\n",
+    "* Ray Core will be the lowest level of abstraction and the most performant.\n",
+    "* Ray Tune will provide fault tolerance, monitoring and integration with experiment management, but will be slower than Ray Core.\n",
```

Comment:
```suggestion
    "* Ray Tune will provide fault tolerance, monitoring and integration with experiment management, but this may add additional overheads for some workloads.\n",
```